draft-ietf-core-resource-directory.md

```diff
@@ -2509,6 +2509,45 @@ have been used to resolve the relative anchor values instead, giving
 
 and analogous records.
 
+## A note on differences between link-format and Link headers
+
+While link-format and Link headers look very similar and are based on the same
+model of typed links, there are some differences between {{RFC6690}} and
+{{RFC5988}} that should be kept in mind when using or implementing a Resource
+Directory:
+
+* There is no percent encoding in link-header documents.
+
+  A link-header document is an UTF-8 encoded string of Unicode characters and
+  does not have percent encoding, while Link headers are practically ASCII
+  strings that use percent encoding for non-ASCII characters, stating the
+  encoding explictly when required.
+
+  For example, while a Link header in a page about a Swedish city might read
+
+      Link: </temperature/Malm%C3%B6>;rel="live-environment-data";
+          title*=UTF-8'en'Live%20temperature%20from%20Malm%C3%B6;
+          other=fields
+
+  a link-format document from the same source might describe the link as
+
+      </temperature/Malmö>;rel="live-environment-data";
+      title*=UTF8'en'Live temperature from Malmö;other=fields
+
+  <!-- The title conversion follows the rule of the text RFC6690 section 2; I
+  doubt that's the intention, though, as it spills at semicolons/commas and
+  does not match the ext-value ABNF. -->
+
+* If the anchor attribute is present, the link target reference is resolved
+  by using the the (resolved) anchor value as Base URI in a link-format
+  document, while in Link headers, it is resolved against the URI of the
+  requested document.
+
+  This is explicit in {{RFC6690}} section 2.1 for link-format, and spellt out
+  in section B.2 of {{I.D.-draft-nottingham-rfc5988bis-08}} <!-- "Parsing a
```

Comment:
Maybe we can make explicit that 6690 was based on 5988, which since has been obsoleted by 8288, but 6690 has not been updated to make use of the simplifications and clarifications offered by 8288.